linera-execution/src/graphql.rs

```diff
@@ -55,7 +55,7 @@ impl Committee {
     }
 }
 
-#[async_graphql::Object]
+#[Object(cache_control(no_cache))]
```

Comment:
Please let's keep `Object` qualified.  The name `Object` without qualification is very non-descriptive, so removing it hurts readability quite a lot.  If we think `async_graphql::Object` is too long, we could at the least import `async_graphql` under a shorter name, like `graphql` or `gql`, to give an indication that this macro is something to do with GraphQL.

Reply:
Ok, I did the replacing on all non-example code.
Maybe there is a case for doing the substitution to other types of async_graphql, but let's stop at that now.